spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -92,41 +92,48 @@ The sample deployment and service YAML files provide configurations for RabbitMQ
 You need to configure only one message broker.
 
 * RabbitMQ
-
++
 Run the following command to start the RabbitMQ service:
-
-
++
+====
 [source,bash]
 ----
 $ kubectl create -f src/kubernetes/rabbitmq/
 ----
-
-
+====
++
 You can use `kubectl get all -l app=rabbitmq` to verify that the deployment, pod, and service resources are running.
 You can use `kubectl delete all -l app=rabbitmq` to clean up afterwards.
 
 * Kafka
-
++
 Run the following command to start the Kafka service:
-
-
++
+====
 [source,bash]
 ----
 $ kubectl create -f src/kubernetes/kafka/
 ----
-
-
+====
++
 You can use `kubectl get all -l app=kafka` to verify that the deployment, pod, and service resources are running.
 You can use `kubectl delete all -l app=kafka` to clean up afterwards.
 
 
 
 [[getting-started-kubernetes-deploy-services-and-data-flow]]
-===== Deploy Services, Skipper and Data Flow
+===== Deploy Services, Skipper, and Data Flow
 
-You must deploy a number of services and the Data Flow server. To do so:
+You must deploy a number of services and the Data Flow server. The following subsections describe how to do so:
 
-1) Deploy MySQL.
+. <<getting-started-kubernetes-deploy-services-and-data-flow-deploy-mysql>>
```

Comment:
Seems like the message broker (Kafka or Rabbit) deployment is missing in the list.

Reply:
I added a note to refer to those tools. We don't need instructions for them, as they have their own sites with instructions.